src/scipp/plotting/__init__.py

```diff
@@ -13,15 +14,11 @@
 except ImportError:
     mpl = None
 
-try:
+# If we are running inside a notebook, then make plot interactive by default.
```

Comment:
@nvaytet Should review the changes in this file.

Reply:
Looks good to me :+1: